lib/api/src/js/thread.rs

```diff
@@ -0,0 +1,44 @@
+use std::{sync::{Arc, mpsc, Mutex}, time::Duration};
+
```

Comment:
This doesn't need to live inside of Wasmer. It can be a side utility/module inside of wasmer-wasi atm (since it's only used there)

Reply:
Or we can move it to WASI

Reply:
This is now moved to WASI